backend/src/Edna/Web/Server.hs

```diff
@@ -50,9 +52,9 @@ translateExceptions action =
 -- | Runs the web server which serves Edna API.
 edna :: Edna ()
 edna = do
+  transact (runPg ensureSchemaIsSetUp)
```

Comment:
AFAIU this `ensureSchemaIsSetUp` doesn't actually ensure anything. We may see that `project` table exists and assume that everything is alright, but it may be a different `project` table, not the one we need (old format or even something completely unrelated that coincidentally was called `project`). AFAIU it only checks the name and nothing else:

> Do not throw an error if a relation with the same name already exists. A notice is issued in this case. Note that there is no guarantee that the existing relation is anything like the one that would have been created.